### PR TITLE
PolRev-centric changes (PR for commenting)

### PR DESCRIPTION
**Pull request by milessa42**

### CODE REVIEW

**Review by innovativedmm — APPROVED** [review]
Thank you!

File: web/index.php [hunk 1]
```diff
@@ -52,13 +58,16 @@
     </div>
   </div>
   <footer class='lato'>
+    // TODO: change the share button href and info
     <div class="fb-share-button" data-href="http://www.bernkit.com/" data-layout="button">
       <a href="javascript:fbShare('http://www.bernkit.com/', 'Fb Share', 'Facebook share popup', 'http://www.bernkit.com/img/fb.png', 520, 350)">Share</a></div>&nbsp;<a href="https://twitter.com/share" class="twitter-share-button"{count} data-url="http://www.bernkit.com" data-text="All the online tools you can use to make Bernie the next president of these United States #feelthebern">Tweet</a>&nbsp;
 <script>
 !function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
-<a href='http://www.berniesanders.com/donate' class='donate-button'>Donate</a>
```
Comment (innovativedmm) [inline web/index.php:59]: https://secure.actblue.com/contribute/page/clean-energy?refcode=rk161207

File: web/index.php [hunk 2]
```diff
@@ -52,13 +58,16 @@
...
+    // TODO: Keep/change donate?
+    <a href='http://www.berniesanders.com/donate' class='donate-button'>Donate</a>
```
Comment (innovativedmm) [inline web/index.php]: https://secure.actblue.com/contribute/page/clean-energy?refcode=em161207

File: web/index.php [hunk 3]
```diff
@@ -52,13 +58,16 @@
...
     <span>&nbsp;&nbsp;&nbsp;
-    &copy; <a href='http://www.reddit.com/r/SandersForPresident' style='display: inline-block'>Grassroots for Sanders 2016</a> &bull; <a href='https://docs.google.com/forms/d/1TMrM4xFagI7SqzSD5qr2VPGUlV9jopisNOnecYq9Cgc/viewform?edit_requested=true' target='_blank' style='display: inline-block'>Join our Call Team</a> &bull; This site is not affiliated with Bernie 2016 campaign. Contact <a href='mailto:rapi@bernie2016events.org' >rapi@bernie2016events.org</a></span>
```
Comment (innovativedmm) [inline web/index.php:61]: <a href="http://political-revolution.com">Political Revolution 2017</a> | <a href="mailto:tech@political-revolution.com">Issues: tech@political-revolution.com</a>

File: web/index.php [hunk 4]
```diff
@@ -3,12 +3,17 @@
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
 
+// TODO: Change title
 <title>Bernie's Volunteer Toolkit - All the online tools you can use to help make Bernie the next president of these United States. Pick and choose, volunteer, feel the bern! Don't see your app?</title>
+// TODO: Change image
 <meta property="og:image" content="http://www.bernkit.com/img/fb.png" />
```
Comment (innovativedmm) [inline web/index.php:9]: I will work on this.

**Review by innovativedmm — APPROVED** [review]
Looks great, I think I got most of the text needed.

**Comment by milessa42** [comment]
Need to redo this PR to the right fork.

**Pull request closed without merging**